src/loaders/db.js

```diff
@@ -0,0 +1,20 @@
+const pg = require('pg');
+const fs = require('fs-extra');
+
+const conString = "postgres://lzxdfpik:bILzLD1yrycRbYZb74fckcKt2tJTiqZF@balarama.db.elephantsql.com:5432/lzxdfpik";
+const client = new pg.Client(conString);
+const sql = fs.readFileSync( "../../init_db.sql", { encoding: "UTF-8" } );
+
+client.connect(function(err) {
+    if(err) {
+        return console.error('could not connect to postgres', err);
+    }
+    client.query(sql, function(err, result) {
+        if(err) {
+            client.end();
+            return console.error('table users already create', err);
+        }
+        console.log(result);
+        client.end();
+    });
+});
```

Comment:
We need here `async/await` contract it's necessary because of our project will use only `async/await` contract.